### ISSUE TITLE
Update README.md with proper install instructions

### ISSUE THREAD

**Issue by dragancla** [reportportal/agent-js-mocha#32]
That `npm install rp-mocha-reporter` seems to install [a module that hasn't been updated in 1 year ](https://www.npmjs.com/package/rp-mocha-reporter) and which does not support ReportPortal v5. Should be `npm install reportportal/agent-js-mocha#master` but, doing so will cause mocha to throw an `"rp-mocha-reporter" reporter not found` error. I'm guessing it's not the correct name or the reporter function is not exported correctly.

**Comment by dragancla** [comment]
The readme altogether is pretty misleading since the repo is called `agent-js-mocha`, the package is called `rp-mocha-reporter` and the name of the reporter, accoding to the readme, is `reporter: 'mocha-rp-reporter'` while, in another issue, someone referred to this as `mocha-js-reporter`

**Comment by ElenaRomanchuk** [comment]
We have just released the new version of mocha reporter as a new package `@reportportal/agent-js-mocha`: https://www.npmjs.com/package/@reportportal/agent-js-mocha.
It's fully compatible with ReportPortal version 5.* and async API v2 and has some additional features.
The package `rp-mocha-reporter` is deprecated now. 
We updated readme and repository name with the new version and the new package.
Thanks.
